packages/app/src/layouts/default.vue

```diff
@@ -110,6 +110,12 @@ const resetErrorAndLoadConfig = (id: string) => {
   }
 }
 
-const renderSidebar = !isRunMode
+const renderSidebar = computed(() => {
+  if (currentRoute.name === 'Specs') {
+    return !isRunMode && query.data.value?.currentProject?.isLoadingConfigFile === false
+  }
+
+  return !isRunMode
```

Comment:
I reverted this change and do not see a difference in behavior.  When does this additional logic come into play?

Reply:
You'll only see it when switching back and forth between E2E and CT using the buttons in the UI. When Cypress re-loads in the new testing type, you'll see the side navigation component flash on screen quickly